remove_association.py

```diff
@@ -0,0 +1,92 @@
+import argparse
```

Comment:
This file doesn't belong in concise. Can we move it to umm-updater maybe?

Reply:
This file is for calling the cmr updater. It imports the necessary methods from it to remove association.
The code for removing the association is in the cmr-updater repo and we are importing it from there.
The l2ss-py has a similar file for calling the cmr-updater.